src/business/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+project(Business)
```

Comment:
small B here

Reply:
think its better to change application to upper case. So we have TWIST::Business and not TWIST::business